desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
Yeah, it's a PR to your branch: https://github.com/damassi/force/pull/1. If you merge it there, it should update this PR. Sorry if this was different from what you suggested, I wasn't sure.

To be honest, I'm not sure I'd want us to get sidetracked too much on a refactor, as much as I'd love for us to take that on. I'll just be happy if we establish good practice going forward. But feel free to ticket if it'll bug you!